jwst/pipeline/calwebb_spec3.py

```diff
@@ -2,7 +2,7 @@
 
 from ..stpipe import Pipeline
 from .. import datamodels
-from ..exp_to_source import exp_to_source
+from ..exp_to_source import multislit_to_container
 
 # step imports
 from ..skymatch import skymatch_step

```

Comment:
That's `--steps.mrs_imatch` (not `mir_imatch`), but yes, other than the typo, that's it.